### PR TITLE
[WIP]BZ#2058433: ADDITIONAL Fix webhook build API endpoint

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 4.6+ (4.6-4.11)
- BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2058433 (https://issues.redhat.com/browse/RHDEVDOCS-3865)
- SME review: @adambkaplan @sferich888 
- QE review: TBD
- Peer reviewer: @jboxman 
- Previews:
  - https://deploy-preview-42441--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/image-streams-manage#images-imagestream-configure_image-streams-managing
  - https://deploy-preview-42441--osdocs.netlify.app/openshift-enterprise/latest/security/container_security/security-container-content#security-integration-reference-example-api-call_security-container-content

### CODE REVIEW

File: modules/service-accounts-using-credentials-inside-a-container.adoc
Comment (rolfedh) [inline modules/service-accounts-using-credentials-inside-a-container.adoc:26]: @adambkaplan Although this file is in the repo, it is currently commented-out, so that it's contents are not visible. Therefore, there is no preview of this content.

We should update the URL anyway, in case someone removes the comment and adds the content back to our documentation.

File: modules/user-agent-overview.adoc
Comment (rolfedh) [inline modules/user-agent-overview.adoc:23]: @adambkaplan This content is similarly not included in the published docs at this time. We should update it in case someone decides to re-add it, eventually.

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Review by adambkaplan — COMMENTED** [review]
/lgtm cancel

File: modules/images-imagestream-configure.adoc
Comment (adambkaplan) [inline modules/images-imagestream-configure.adoc:27]: This is incorrect - this _might_ be correct, though I am not sure
```suggestion
  selflink: /apis/core/v1/namespaces/test/imagestreams/origin-ruby-sample
```

File: modules/security-container-content-external-scanning.adoc
Comment (adambkaplan) [inline modules/security-container-content-external-scanning.adoc:239]: cc @dmage 

```suggestion
  https://<openshift_server>:8443/apis/image.openshift.io/v1/images/<image_id> \
```

File: modules/service-accounts-using-credentials-inside-a-container.adoc
Comment (adambkaplan) [inline modules/service-accounts-using-credentials-inside-a-container.adoc:29]: I think users are in the core API namespace, cc @sttts @deads2k:
```suggestion
    "https://openshift.default.svc.cluster.local/apis/core/v1/users/~" \
```

File: modules/service-accounts-using-credentials-inside-a-container.adoc
Comment (adambkaplan) [inline modules/service-accounts-using-credentials-inside-a-container.adoc:36]: Ditto, I think this is in core:
```suggestion
  selflink: "/apis/core/v1/users/system:serviceaccount:top-secret:robot"
```

File: modules/user-agent-overview.adoc
Comment (adambkaplan) [inline modules/user-agent-overview.adoc:26]: Not sure how this is done outside of `/oapi` - this content may no longer be relevant.

cc @sttts

File: modules/user-agent-overview.adoc
Comment (adambkaplan) [inline modules/user-agent-overview.adoc:30]: Similarly, I am not sure if this capability exists outside of `/oapi`.

**Review by adambkaplan — COMMENTED** [review]
/hold

A lot of these changes touch things that may not exist any more. Also need to verify what the endpoint is for core APIs.

**Comment by rolfedh** [comment]
Hi, @sttts and @deads2k. Would you please review the updates to these few lines of documentation?

**Comment by jboxman** [comment]
@rolfedh, LGTM if the API endpoints are good.

**Comment by jboxman-rh** [comment]
@rolfedh, LGTM if the API endpoints are good.

File: modules/security-container-content-external-scanning.adoc
Comment (xiuwang) [inline modules/security-container-content-external-scanning.adoc:239]: Should we use 6443 port, e.g. `https://<openshift_server>:6443`

Comment (wangke19) [reply modules/security-container-content-external-scanning.adoc:239]: Yes, 6443 is used for kube-apiserver, 8443 is for openshift-apiserver

File: modules/service-accounts-using-credentials-inside-a-container.adoc
Comment (wangke19) [inline modules/service-accounts-using-credentials-inside-a-container.adoc:29]: AFAIK， core is a api group name, not the path of the api. 
`oc get --raw "/apis" | grep '/core'`
no results
```
$ oc get --raw "/" | grep user
    "/apis/user.openshift.io",
    "/apis/user.openshift.io/v1",

For example,
$ token=$(oc whoami -t)
$ curl -X GET https://<openshift-kube-apiserver>:6443/apis/user.openshift.io/v1/users/~ --header "Authorization: Bearer $token" --insecure
{
  "kind": "User",
  "apiVersion": "user.openshift.io/v1",
  "metadata": {
    "name": "kube:admin",
    "selfLink": "/apis/user.openshift.io/v1/users/kube%3Aadmin",
    "creationTimestamp": null
  },
  "identities": null,
  "groups": [
    "system:authenticated",
    "system:cluster-admins"
  ]
}
```

File: modules/service-accounts-using-credentials-inside-a-container.adoc
Comment (rolfedh) [inline modules/service-accounts-using-credentials-inside-a-container.adoc:29]: @kalexand-rh Based on your work in https://github.com/openshift/openshift-docs/pull/13010/files, can you help me find the SME can suggest the correct replacement for this URL?

Comment (kalexand-rh) [reply modules/service-accounts-using-credentials-inside-a-container.adoc:29]: @bergerhoffer, this looks like auth. Do you know?

Comment (s-urbaniak) [reply modules/service-accounts-using-credentials-inside-a-container.adoc:29]: at first eye sight this looks correct to me. i'd like to verify on a cluster though.

Comment (ibihim) [reply modules/service-accounts-using-credentials-inside-a-container.adoc:29]: As far as I can tell from traversing through the API on an OpenShift 4.10 cluster it looks like so:

```bash
curl --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt "https://openshift.default.svc.cluster.local/apis/user.openhift.io/v1/users/~" -H "Authorization: Bearer $TOKEN"
```

and the output is like so:

```json
{
  "kind": "User",
  "apiVersion": "user.openshift.io/v1",
  "metadata": {
    "name": "system:serviceaccount:openshift-authentication:oauth-openshift",
    "creationTimestamp": null
  },
  "groups": [
    "system:authenticated",
    "system:serviceaccounts",
    "system:serviceaccounts:openshift-authentication"
  ]
}
```

**Pull request closed without merging**
